packages/autorest.typescript/test/smoke/swagger/anomaly-detector-rest.md

```diff
@@ -11,7 +11,7 @@ description: AnomalyDetector Rest Client
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../
 source-code-folder-path: ./src/
-require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/main/specification/cognitiveservices/data-plane/AnomalyDetector/readme.md
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/021e5ab6ec840d651d32a2c6bdf832800cdf8eb6/specification/cognitiveservices/data-plane/AnomalyDetector/readme.md
```

Comment:
This main cause the change of anomaly detector smoke test change. fix it so that it won't cause problems in the future.